src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -267,10 +266,11 @@ public boolean isNotSimpleTranslocation() {
     }
 
     Tuple2<SimpleInterval, SimpleInterval> getCoordSortedReferenceSpans() {
-        if (involvesRefPositionSwitch(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig))
-            return new Tuple2<>(regionWithHigherCoordOnContig.referenceSpan, regionWithLowerCoordOnContig.referenceSpan);
-        else
+        if (isForwardStrandRepresentation) {
```

Comment:
used in two places: `BC.iniForInsDel()` and `NARL.leftJustifyBreakpoints()`.
The first use is trivial as it is type specific, whereas for the 2nd use in NARL, we need to be careful since we are considering expanding the NARL to cover cases of ref order switch.